nodejs/sample-apps/aws-sdk/deploy/main.tf

```diff
@@ -1,3 +1,7 @@
+locals {
+  collector_layer_arn = "arn:aws:lambda:${data.aws_region.current.name}:901920570463:layer:aws-otel-nodejs-ver-0-23-0:1"
```

Comment:
Hi @kbkk - this layer is the AWS vendor distro, so we don't add it to the examples in this repo. We have sample apps using it in aws-otel-lambda.

We currently haven't published a public layer for this repo so can't make this change unfortunately.

Reply:
Ah, I understand. Sorry for the confusion. Closing